coreos-base/oem-packet/oem-packet-0.0.5.ebuild

```diff
@@ -16,14 +16,11 @@ S="${WORKDIR}"
 
 src_prepare() {
 	sed -e "s\\@@OEM_VERSION_ID@@\\${PVR}\\g" \
-	    "${FILESDIR}/cloud-config.yml" > "${T}/cloud-config.yml" || die
+		"${FILESDIR}/oem-release" > "${T}/oem-release" || die
 }
 
 src_install() {
-	into "/usr/share/oem"
-	dobin "${FILESDIR}/phone-home.sh"
```

Comment:
I think you created a PR from a branch in this repo, but you pushed the update to your own forked repo.  That version looks okay, but does that now need an Ignition update?